dangerfile.js

```diff
@@ -38,7 +38,7 @@ const RELEASE_CHANNEL =
 const artifactsJobName =
   process.env.RELEASE_CHANNEL === 'experimental'
     ? 'process_artifacts_experimental'
-    : 'process_artifacts';
+    : 'process_artifacts_stable';
```

Comment:
Needs diffs to sync scripts or a note in the group for the next person.

Reply:
Reverting this to avoid the thrash. This is not a command you run anyway.